packages/server/src/controllers/UsersController.js

```diff
@@ -0,0 +1,49 @@
+import Models from '../models'
+import UsersValidator from '../validators/UsersValidator'
+import { toChecksumAddress } from 'web3-utils'
+
+const { User, UserAddress } = Models
+
+export default {
+  async exists(request, response, next) {
+    try {
+      const errors = await UsersValidator.validateForCreate(request.query)
+      return (errors.length === 0) ? response.status(200) : response.status(400).send({ errors })
```

Comment:
I might be missing context, but it seems kind of weird we have a pattern where we short circuit with a `400` inside, but if the method throws, we pass the error along to `next()`.

Reply:
I was trying to follow [this guide](https://expressjs.com/en/guide/error-handling.html), but maybe there is a better way to handle errors

Reply:
Yes, I'm not entirely familiar with how the middleware works when you use `response.send()` like this, but I assume it'll short circuit?

Anyhow, if we need to, we can refactor how we do this input validation in a different PR.